.github/workflows/release.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.19
+          go-version: 1.21
```

Comment:
Make this read the go version from `go.mod` file instead?

Reply:
Correct, I will fix it